web-components/src/api/folksonomy.ts

```diff
@@ -16,6 +17,118 @@ const getApiUrl = (path: string) => {
   return `${folksonomyConfiguration.getItem("apiUrl")}${path}`
 }
 
+/**
+ * Get stored token from localStorage
+ * @returns {string | null}
+ */
+function getStoredToken(): string | null {
+  return localStorage.getItem('bearer')
+}
+
+/**
+ * Save token to localStorage
+ * @param token
+ */
+function saveTokenToStorage(token: string) {
+  localStorage.setItem('bearer', token)
+  localStorage.setItem('date', new Date().getTime().toString())
```

Comment:
There also you have a tremendous chance of name clash… Please change for folksonomy-token-date.